src/PropertyInfo/Extractors/PhpDocExtractor.php

```diff
@@ -102,7 +102,8 @@ public function extractTypes(\ReflectionProperty $reflectionProperty)
      */
     private function getFileReflector(\ReflectionClass $reflectionClass)
     {
-        if (!($fileName = $reflectionClass->getFileName())) {
+        if (!($fileName = $reflectionClass->getFileName())
+        or !in_array(pathinfo($fileName, PATHINFO_EXTENSION), ['php', 'phtml'])) {
```

Comment:
I have a personal preference for `and` and `or` specifically because they have lower precedence. It leads to bugs when you're not aware of it but for conformity's sake I'll flip them the other way round.

`*.hh` files are one example but PhpDoc fails to parse `*.php` files which have PHP7 syntax inside -- what about those? I could name those files `*.php7` for now and exclude them as such...